silver/utils/date_utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2015 Presslabs SRL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+ONE_MONTH = datetime.timedelta(weeks=4)
```

Comment:
This doesn't look right to me. 4 weeks != 1 month
As we are already using python-dateutil in the project, have a look at this examples (especially the ~3rd one): https://dateutil.readthedocs.org/en/latest/examples.html#relativedelta-examples